CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
-# 0.6.0
+# 0.7.0
+
+## New Features
+
+* First version of **GitLab** support. (Implementation is not completed yet.) (@lbarcziova)
+* Add method for loading services from dictionary. (@lachmanfrantisek)
+* Release objects has a method for editing. (@lbarcziova)
+* Function for getting all commits from specific PR. (@phracek)
+
+## Fixes
 
+* Fix creating pull-request from fork to upstream on new versions of Pagure. (@lachmanfrantisek)
+* Use web url in Pagure issue. (@dustymabe)
+
+## Minor
```

Comment:
I'd say that some items in this section are completely unclear for an average reader - just sayin' :P you don't need to mention every commit in the changelog

Reply:
I've already removed nearly 50 entries..;) The rest looks relevant to me.

Reply:
What do you want to remove? I am proud that we have all these changes...;-)

Maybe we can remove this one:
>* Make the pagure service mapping more general. (@lachmanfrantisek)

Reply:
leave it as it is, my comment was for the future changelog entries to be honest